src/classes/ADDR_Addresses_UTIL.cls

```diff
@@ -416,17 +416,17 @@ public with sharing class ADDR_Addresses_UTIL {
 
         // If State & Country Picklists are enabled in the org, build maps of the
         // valid Labels and Codes for each field to use for validation
-        map<String, Schema.SObjectField> acctFields = Account.getSobjectType().getDescribe().fields.getMap();
+        Map<String, Schema.SObjectField> acctFields = Account.getSobjectType().getDescribe().fields.getMap();
```

Comment:
I have changed the data structure class capitalization in a number of places. Please ignore.